packages/devtools_app/lib/src/screens/profiler/profiler_screen_controller.dart

```diff
@@ -82,6 +82,19 @@ class ProfilerScreenController extends DisposableController
         ProfilerScreen.id,
         createData: (json) => CpuProfileData.fromJson(json),
         shouldLoad: (data) => !data.isEmpty,
+        loadData: (CpuProfileData data) async {
```

Comment:
leave this in a separate method and then pass the method as a tearoff
`loadData: processOfflineData,`